middle/isq.owl

```diff
@@ -46,29 +46,6 @@ email: emanuele.ghedini@unibo.it</rdfs:comment>
     
```

Comment:
The changes in isq.owl will most likely make a lot of conflicts with PR93. I will suggest that you exclude isq.owl from this PR.